themes/default/content/docs/concepts/resources/names.md

```diff
@@ -219,7 +219,7 @@ let role = new aws.iam.Role("my-role", {
 
 ```python
 role = iam.Role('my-role', {
```

Comment:
The same thing here, I believe it is these wrapping curly braces that are the problem.

```python
role = iam.Role('my-role',
    name=f'my-role-{ pulumi.get_project() }-{ pulumi.get_stack() }'), 
    opts=ResourceOptions(delete_before_replace=True)
)
```

Attempting to deploy with a colon in Python actually gives me an error when running `pulumi up`.

I do believe that this is an indicator that the examples on this page can be improved, so I will be creating a separate issue for that.